lib/rubocop/cop/style/arguments_forwarding.rb

```diff
@@ -186,46 +187,36 @@ def classification_and_forwards(def_node, send_node, referenced_lvars, forwardab
 
         def register_forward_args_offense(def_arguments_or_send, rest_arg_or_splat)
           add_offense(rest_arg_or_splat, message: ARGS_MSG) do |corrector|
-            unless parentheses?(def_arguments_or_send)
-              add_parentheses(def_arguments_or_send, corrector)
-            end
```

Comment:
I realised I could reuse this functionality across `register_forward_args_offense`, `register_forward_kwargs_offense` and the new method `register_forward_all_offense` (which replaces the two separate `register_forward_all_offense_on_forwarding_method` and `register_forward_all_offense_on_method_def`)